actions/signing-event/action.yml

```
@@ -35,12 +35,14 @@ runs:
 
           issue = 0
           const repo = context.repo.owner + "/" + context.repo.repo
-          const issues = await github.rest.search.issuesAndPullRequests({
-            q: "label:" + process.env.GITHUB_REF_NAME + "+state:open+type:issue+repo:" + repo,
+          const issues = await github.rest.issues.listForRepo({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            labels: process.env.GITHUB_REF_NAME,
```

I believe this works but using an array here would be more correct I think

```suggestion
            labels: [process.env.GITHUB_REF_NAME],
```